core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -524,10 +530,11 @@ public User createAccount(String userName, String password) throws IOException {
     }
 
     /**
-     * Creates a new user account by registering a JBCrypt Hashed password with the user.
+     * Creates a new user account by registering a Hashed password with the user.
      *
      * @param userName The user's name
-     * @param hashedPassword A hashed password, must begin with {@code #jbcrypt:}
+     * @param hashedPassword A hashed password, must begin with {@code getPasswordHeader()}
+     * @see #getPasswordHeader()
      */
     public User createAccountWithHashedPassword(String userName, String hashedPassword) throws IOException {
         if (!PASSWORD_ENCODER.isPasswordHashed(hashedPassword)) {
```

Comment:
Next line could probably be enhanced to identify a wrong hash format and if so, state that.

Reply:
f37bcc31de0a041a2f1782fa3bf253701d04ebcd